<?xml version="1.0" ?>
<launch>

  <node name="pcl_cloud_processing_service" pkg="temporal_grasp_ros" type="pcl_service" output="screen"/>
  <node name="temporal_grasp_planner_node" pkg="temporal_grasp_ros" type="temporal_grasp_node.py" output="screen"/> 
  <node name="grasp_execution_gui" pkg="temporal_grasp_ros" type="execute_grasp_gui.py" output="screen"/>

</launch>
